src/assets/PythonCopyFileWithShutil/python-copy-paste.md

```diff
@@ -0,0 +1,113 @@
+---
+title: Ways to copy file with shutil module in Python
+authorName: Salipa Gurung
+authorAvatar: https://avatars.githubusercontent.com/u/53458341?v=4
+authorLink: https://github.com/Salipa-Gurung
+createdAt: Feb 27, 2024
+tags: python, copy-paste, shutil
+banner: https://blog.jankaritech.com/src/assets/PythonCopyFileWithShutil/images/python_copy_file.png
+---
+
+Python provides many built-in modules to copy and paste a file. We'll look into ways to copy file using `shutil` module in this blog post.
+
+The standard utility module known as shutil is an abbreviation for shell utilities. Numerous functions in the Python shutil module helps with high-level file and directory operations. Using the shutil module, there are four common methods that provide an easy and efficient way to copy and paste a file:
+
+### 1. shutil.copy() method
+
+```py
+shutil.copy(source, destination, *, follow_symlinks=True)
+```
+
+This method copies the file of source to the directory or file specified in the destination and returns file's destination.
+
```

Comment:
If you write the following section in the bullet point style, it would be nice to have an introductory sentence, something like "This method is characterized by the following points" or "Let's have a look at the parameters of the method"